README.md

```diff
@@ -55,13 +54,15 @@ In the same HTML document as this is defined, then add a script tag referencing
 ### Mandatory Attributes:
 - id : value=<string> - Div needs ID to be uniquely identified by the library. Must be unique (Per HTML standards), but is not limited besides that.
 - class : value="CarPicsSpinner" - The library gets elements by class, and sets up a spinner in any div with classvalue = "CarPicsSpinner".
-- CarPicsUrl : value=<string, resolvable URL> - The library fetches from this url. Expects a JSON array of objects with "src" parameter. Does not discriminate on endpoint – will request from any url, and attempt to make images from the return. Allows full flexibility in delivery. 
+- vin : value="<vin string>" - The vehicle vin, used to lookup the spinner resource. 

```

Comment:
Add that "This information should be available in the system that generates that vehicle page."